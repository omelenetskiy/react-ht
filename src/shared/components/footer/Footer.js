import React from 'react';
import { Link } from 'react-router-dom';

import { appTitle } from '../../../config/moviesApp';
import StyledFooter from '../../styled/footer';

const Footer = () => (
  <StyledFooter>
    <div className="footer">
      <Link to="/404">{appTitle}</Link>
    </div>
  </StyledFooter>
);

export default Footer;
