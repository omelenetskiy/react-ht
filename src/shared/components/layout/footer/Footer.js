import React from "react";
import { Link } from "react-router-dom";

const Footer = () => (
  <footer>
    <Link to="/404">Netflixroulette</Link>
  </footer>
);

export default Footer;
