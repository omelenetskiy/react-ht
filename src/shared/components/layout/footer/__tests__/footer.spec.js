import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import Footer from '../Footer';

describe('Footer', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Footer />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
