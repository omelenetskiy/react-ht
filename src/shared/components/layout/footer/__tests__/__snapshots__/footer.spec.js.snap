// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<footer___default>
  <Link
    replace={false}
    to="/"
  >
    Netflixroulette
  </Link>
</footer___default>
`;
